app/views/landing_page/blocks/_main_navigation.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  add_view_stylesheet("landing_page/main-navigation")
+%>
+
+<div class="app-b-main-nav" data-module="app-b-main-navigation">
+  <p class="app-b-main-nav__heading govuk-heading-s"><%= block.data["title"]%></p>
+  <button class="app-b-main-nav__button js-app-b-main-nav__button govuk-link govuk-link--no-underline" aria-expanded="false" aria-controls="app-b-main-nav__nav" type="button">Menu<span class="app-b-main-nav__icon" aria-hidden="true"></span></button>
+
+  <nav id="app-b-main-nav__nav">
+    <ul class="app-b-main-nav__ul">
+      <% block.data["links"].each do |link| %>
+        <%
+          li_classes = "app-b-main-nav__listitem"
+          li_aria = { current: true } if link["active"]
+        %>
+        <%= tag.li(class: li_classes, aria: li_aria) do %>
+          <% if link["active"] %>
```

Comment:
I think this bit down to around line 29 could be reduplicated a little, something like

```
          <%
            link_classes = "app-b-main-nav__link govuk-link govuk-link--no-underline"
            link_classes << " app-b-main-nav__link--active" if link["active"]
          %>
          <%= tag.a(href: link["link"], class: link_classes) do %>
            <span class="app-b-main-nav__mobile-border"></span>
            <%= link["label"] %>
          <% end %>
```

Reply:
Thanks, have cleaned up the code and fixed the bug, good spot 👍